app/models/vm_or_template.rb

```diff
@@ -1755,6 +1755,10 @@ def parent_resource
     parent
   end
 
+  def self.display_name(number = 1)
+    n_('VM or Template', 'VMs or Templates', number)
```

Comment:
Is this descriptive enough?  It doesn't include "Instance(s)" and "Template(s)".  That being said, is it used?